packages/core/src/icon/index.ts

```diff
@@ -6,6 +6,7 @@
 
 export * from './icon.element.js';
 export * from './icon.service.js';
+export { updateIconSizeStyleOrClassnames } from './utils/icon.classnames.js';
```

Comment:
I was exporting this when the circular progress was reusing the same logic as icons for sizing. I'm not doing that anymore so this is probably unnecessary.